web3/contract.py

```diff
@@ -1678,6 +1657,47 @@ def getLogs(
         # Convert raw binary data to Python proxy objects as described by ABI
         return tuple(get_event_data(self.w3.codec, abi, entry) for entry in logs)
 
+    @combomethod
+    def create_filter(
+        self,
+        *,  # PEP 3102
+        argument_filters: Optional[Dict[str, Any]] = None,
+        fromBlock: Optional[BlockIdentifier] = None,
+        toBlock: BlockIdentifier = "latest",
+        address: Optional[ChecksumAddress] = None,
+        topics: Optional[Sequence[Any]] = None,
+    ) -> LogFilter:
+        """
+        Create filter object that tracks logs emitted by this contract event.
+        # optional --- update the params descriptions here or remove this line
+        """
+        filter_builder = EventFilterBuilder(self._get_event_abi(), self.w3.codec)
+        self._set_up_filter_builder(
+            argument_filters,
+            fromBlock,
```

Comment:
snake case this if this comment isn't resolved after rebase with snake casing PR.